src/main/java/org/openpnp/machine/reference/feeder/wizards/ReferenceStripFeederConfigurationWizard.java

```diff
@@ -587,7 +587,7 @@ public void run() {
             return holeLocations;
         }
         finally {
-            pipeline.release();
+            releaseCvPipeline(pipeline);
```

Comment:
I don't think any of this is needed, but I'm open to being convinced I'm wrong. 

Since we're cloning the pipeline, there shouldn't be any references to it held past the time it's used. Once those references are orphaned the entire pipeline, including the properties map and the references to the feeder, camera, etc. will be collected by the GC.

I do agree that the configuration listener stuff below is needed for now, since the configuration listener will keep the camera around. I'm willing to merge that bit temporarily, but long term I want to see two other changes:

1. VisionProvider is on it's way out. It's only used by ReferenceDragFeeder now so if we convert ReferenceDragFeeder either to a pipeline to just use FluentCv we can drop the entire VisionProvider interface and everything that goes along with it.
2. I'm not crazy about the creation of a Camera every time we call the pipeline. I realize it's just a wrapper for an image, but it feels hacky and I think we'd be better off just injecting the image somehow.